content/post/articles/Become-A-Software-Developer.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Become a Software Developer"
+date: 2021-11-30T15:32:19+01:00
+images: ["/Become-A-Software-Developer/shaba-rex.png"]
+categories: ["Software Developer"]
+tags: ["Software Developer","BootCamps","Self-taught Developer"]
+authors: ["Shaba-Rex"]
+---
+
+{{<figure src="/Become-A-Software-Developer/viva the it.jpeg"  alt="Software Engineering is Cool">}}
```

Comment:
Are you sure this picture is free of right? Moreover, you should definitely consider using one that doesn't have a watermark.

Reply:
Ok i got it.